lego/apps/events/tasks.py

```diff
@@ -226,9 +236,19 @@ def stripe_webhook_event(self, event_id, event_type, logger_context=None):
             raise WebhookDidNotFindRegistration(event_id, metadata)
         registration.charge_id = serializer.data["id"]
         registration.charge_amount = serializer.data["amount"]
-        registration.charge_amount_refunded = serializer.data["amount_refunded"]
+        registration.charge_amount_refunded = (
+            serializer.data["amount_refunded"]
+            if "amount_refunded" in serializer.data
+            else 0
+        )
         registration.charge_status = serializer.data["status"]
         registration.save()
+        if event_type == "payment_intent.succeeded":
```

Comment:
Hmm :thinking:

Reply:
Should we notify user or retry if event_type is not succeeded?

Reply:
This is handled in the frontend, as stripe serves the errors and we handle retrying there. The webhooks are really just for setting the correct values in the db.